{"failedTests":[{"testName":"parallel/test-runner-run-watch","timestamp":1725576481772,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #30002","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/30002/","buildNumber":30002,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"4cd82f1a179994f9ed5cb2274bee651d45b3b94c","callStack":[{"upstreamBuild":65582,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74028,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62022,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54651"}],"tap":{"ok":false,"name":"parallel/test-runner-run-watch","id":896,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout"},"time":121624.95,"fullname":"parallel/test-runner-run-watch","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725576481772,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #30002","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/30002/","buildNumber":30002,"builtOn":"test-rackspace-win2022_vs2022-x64-4","nodeVersion":"23.0.0","commitHash":"4cd82f1a179994f9ed5cb2274bee651d45b3b94c","callStack":[{"upstreamBuild":65582,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74028,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62022,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54651"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":110,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4621.5567ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4748.3867ms)\n  ✖ async-hooks\\test-async-local-storage-async-functions.js (4924.0732ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-functions.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 2)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5107.1768ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (4804.0091ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4610.7578ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (5610.5414ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (5012.4091ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (6229.5544ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (5490.1056ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (5354.9494ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (5194.7532ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5417.5734ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5407.855ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (5716.4568ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (6035.051ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5728.746ms)\n  ✔ parallel\\test-async-local-storage-bind.js (5719.4947ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4707.8418ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (5773.6812ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (5614.7286ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5119.7449ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (5702.9931ms)\n▶ AsyncContextFrame (6332.7926ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 6350.2929\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-async-functions.js (4924.0732ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-functions.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 2)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":6946.602,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}